contracts/epochs/FlowEpoch.cdc

```diff
@@ -674,24 +674,36 @@ pub contract FlowEpoch {
 
     /// Borrow a reference to the FlowIDTableStaking Admin resource
     access(contract) fun borrowStakingAdmin(): &FlowIDTableStaking.Admin {
-        let adminRef = self.account.borrow<&FlowIDTableStaking.Admin>(from: FlowIDTableStaking.StakingAdminStoragePath)
-            ?? panic("Could not borrow staking admin")
+        let adminCapability = self.account.copy<Capability>(from: FlowIDTableStaking.StakingAdminStoragePath)
+            ?? panic("Could not get capability from account storage")
+
+        // borrow a reference to the staking admin object
+        let adminRef = adminCapability.borrow<&FlowIDTableStaking.Admin{FlowIDTableStaking.EpochOperations}>()
+            ?? panic("Could not borrow reference to staking admin")
 
         return adminRef
     }
 
     /// Borrow a reference to the ClusterQCs Admin resource
     access(contract) fun borrowClusterQCAdmin(): &FlowClusterQC.Admin {
-        let adminRef = self.account.borrow<&FlowClusterQC.Admin>(from: FlowClusterQC.AdminStoragePath)
-            ?? panic("Could not borrow qc admin")
+        let adminCapability = self.account.copy<Capability>(from: FlowClusterQC.AdminStoragePath)
```

Comment:
Should we define a separate path for the `EpochOperations` capability? My guess is it would be better to have separate paths for capabilities vs resources, in case they are stored in the same account.

Reply:
Based on our conversations, there is only ever going to be one of each epoch operations resource and one of each capability and they should never move again after the upcoming change, so I think it makes sense to re-use the path

Reply:
I agree that on Mainnet the paths won't conflict. I was also thinking about making sure we have a functional deployment for new instances of the contract (for example for integration tests, or new networks) in https://github.com/onflow/flow-core-contracts/pull/367#issuecomment-1533313574. 

I feel like the simplest way to accomplish this is to store both the admin resources and capabilities to the same account, by default, which would require specifying distinct path names for the new capabilities.